### DISCUSSION TITLE
Create body from capture

### DISCUSSION THREAD

**Discussion by schophil** [Orange-OpenSource/hurl#2849] [Q&A] [answered]
I want to use the body of a GET in the PUT of another request. The services respond JSON.

Something like this:

```
GET https://localhost/formconfigs/{{form_id}}
Authorization: Bearer {{access_token}}
HTTP 200
[Captures]
form_body: body

PUT http://localhost/formconfigs/{{form_id}}
Content-Type: application/json
Authorization: Bearer {{access_token}}
{{form_body}}
HTTP 200
```

But this doesn't seem to work. I get:

```
error: Invalid JSON
  --> /tmp/test.hurl:27:3
   |
   | PUT http://localhost/{{form_id}}
   | ...
27 | {{form_body}}
   |   ^^^^^^^^^ actual value is <{...}>
```

Is there a way to make this work?

**Comment by schophil** [comment, marked as answer]
Ah I think I found the solution, this seems to to do the trick:

````
PUT http://localhost/formconfigs/{{form_id}}
Content-Type: application/json
Authorization: Bearer {{access_token}}
```json
{{form_body}}
```
HTTP 200
````

Cna anyone conform this is the right way to do it?

**Reply by jcamiel** [reply]
Hi @schophil 

This is the right way to do this see #2233 and my comment https://github.com/Orange-OpenSource/hurl/issues/2233#issuecomment-1854016062

Note that with the multiline string syntax (` ```json`), you can omit to send the `Content-Type` header as Hurl infers it from the `json` "language hint":

~~~
PUT http://localhost/formconfigs/{{form_id}}
Authorization: Bearer {{access_token}}
```json
{{form_body}}
```
HTTP 200
~~~

**Reply by schophil** [reply]
Ok thanks a lot!
